webpack.dev.js

```diff
@@ -0,0 +1,28 @@
+const merge = require('webpack-merge');
+const config = require('./webpack.config.js');
+const WriteFilePlugin = require('write-file-webpack-plugin');
+
+module.exports = merge(config, {
+  mode: 'development',
+  devtool: 'source-map',
+  devServer: {
+    proxy: {
+      '/extensions': {
+        target: 'http://localhost:3001',
+        pathRewrite: { '^/extensions': '/public/extensions' }
+      }
+    },
+    port: 3001
+  },
+  plugins: [
+    ...config.plugins,
+    new WriteFilePlugin()
```

Comment:
Why would we need to write the bundle to disk during development?

Reply:
Because `webpack` is watching changes that occur in the file system, and not changes in memory. In this case, whenever a new change to the `snjs` module is made, the bundle is update. Immediately after, the bundle in the `web` app is updated too.

The same applies when any change is made within the `web` app. Although, it will not update `snjs` module if no new changes are detected.